src/components/Calculator/__snapshots__/Calculator.test.js.snap

```diff
@@ -0,0 +1,154 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
- [ ] There should be the usage of Jest snapshots for testing all the components. Currently I can see snapshots only for Calculator